src/Aquarius.Client/TimeSeries/Client/IAquariusClient.cs

```diff
@@ -47,5 +47,7 @@ public interface IAquariusClient : IDisposable
 
         [Obsolete("Sessions will now re-authenticate automatically if they expire.")]
         ScopeAction SessionKeepAlive();
+
+        void UpdateBearerToken(string accessToken);
```

Comment:
Wasn't sure about adding this since it breaks the auth abstraction a bit, but otherwise it is pretty tedious to update the bearer token as you can see in the implementation.

Reply:
Not easy to make it fit nicely. The SDK wasn't really designed for any other auth types than credentials.